e2etestrunner-collector/smoke_test.go

```diff
@@ -0,0 +1,134 @@
+//go:build e2ecollector
+
+package e2etestrunner_collector
+
+import (
+	"cloud.google.com/go/logging"
+	"cloud.google.com/go/logging/logadmin"
+	monitoring "cloud.google.com/go/monitoring/apiv3/v2"
+	monitoringpb "cloud.google.com/go/monitoring/apiv3/v2/monitoringpb"
+	"context"
+	"errors"
+	"fmt"
+	"google.golang.org/api/cloudtrace/v1"
+	"google.golang.org/api/iterator"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	representativeMetric = "workload.googleapis.com/otelcol_exporter_sent_metric_points"
+	queryMaxAttempts     = 3
+	labelName            = "otelcol_google_e2e"
+)
+
+func TestMetrics(t *testing.T) {
+	ctx := context.Background()
+	c, err := monitoring.NewMetricClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer c.Close()
+
+	tsList := make([]*monitoringpb.TimeSeries, 0)
+	for attempt := 1; attempt <= queryMaxAttempts; attempt++ {
+		window := 10 * time.Minute
+		now := time.Now()
+		start := timestamppb.New(now.Add(-window))
+		end := timestamppb.New(now)
+		fmt.Println(fmt.Sprintf("resource.type = %q", resourceType))
+		filters := []string{
+			fmt.Sprintf("metric.type = %q", representativeMetric),
+			fmt.Sprintf("resource.type = %q", resourceType),
+			fmt.Sprintf("metric.labels.otelcol_google_e2e = %s", args.TestRunID),
+		}
+
+		req := &monitoringpb.ListTimeSeriesRequest{
+			Name:   "projects/" + args.ProjectID,
+			Filter: strings.Join(filters, " AND "),
+			Interval: &monitoringpb.TimeInterval{
+				EndTime:   end,
+				StartTime: start,
+			},
+			View: monitoringpb.ListTimeSeriesRequest_FULL,
+		}
+
+		it := c.ListTimeSeries(ctx, req)
```

Comment:
`it` might be a bit of an odd name for readers who don't know that what `ListTimeSeries` returns is an "iterator". Maybe `tsIter` would be better (to be similar to `tsList`)?